src/components/interaction-designer/block-editors/UploadMonitor.vue

```diff
@@ -4,7 +4,7 @@
       <h6>{{ 'trees.uploading' | trans }}&hellip; {{ upload.file.name }}</h6>
 
       <div class="progress">
-        <progress-bar :now="progress.toFixed(1) * 1" label type="default" striped animated/>
+        <b-progress :value="progress * 1" show-value animated class="mb-3"/>
```

Comment:
Ok! @jory-viamo Was the removal of `.toFixed(1)` intentional here?

Reply:
Check out the docs: https://bootstrap-vue.org/docs/components/progress#comp-ref-b-progress
You can optionally provide prop `precision` (default 0) to control decimals. By default, it will show the nearest whole number.